src/App.tsx

```diff
@@ -1,101 +1,16 @@
 /* eslint-disable max-len */
 import React from 'react';
+import { MoviesList } from './components/MoviesList';
+import movies from './api/movies.json';
 
 import './App.scss';
 // import moviesFromServer from './api/movies.json';
```

Comment:
i don't see the file 'App.scss' in your tree. Did you push it here? 
Through that, your demo looks different -> https://vladimir-dublya.github.io/react_movies-list/